.github/workflows/tags.yml

```diff
@@ -1,15 +1,17 @@
+---
+name: Deploy to Homebrew
 on:
   push:
-    tags: 'v*'
+    tags: 'v*' # Push events to tags matching v*
```

Comment:
Honestly, there are no "tags matching `v*`" for v2.221.1 but the formula is still bumped FWIW;] the brew bumps are being done elsewhere:
https://github.com/fastlane/fastlane/blob/5aa842986cbaf5298623e340241623d73901f35f/fastlane/Fastfile#L435
so this may be completely superfluous…(?)